src/back/project/src/api/di-plugin-param.cpp

```diff
@@ -57,7 +58,9 @@ formats::json::Value DiPluginParam::Post(
 	const formats::json::Value& body,
 	formats::json::ValueBuilder& res) const
 {
-	const auto diPluginParam = body.As<model::DiPluginParam>();
+	auto diPluginParam = body.As<model::DiPluginParam>();
+	auto spaceId = utils::BoostUuidFromString(req.GetHeader("X-Space-Id"));
+	diPluginParam.spaceId = spaceId;
```

Comment:
Давай сразу присваивать значение модели:

```suggestion
	diPluginParam.spaceId = utils::BoostUuidFromString(req.GetHeader("X-Space-Id"));
```

Reply:
ОК (уже не актуально, так как все эти места ушли после рефакторинга на шаблоны).